test/utils/DomTest.ts

```diff
@@ -364,7 +364,7 @@ export function DomTests() {
         expect(spy).toHaveBeenCalledTimes(1);
       });
 
-      it('using off should work properly', () => {
+      it('after calling off the event handler should not be called', () => {
```

Comment:
Possible to tweak the description slightly for clarity: `when calling 'off', the event handler should no longer be triggered`